src/modules/products/server/procedures.ts

```
@@ -0,0 +1,62 @@
+                depth: 1,
+                pagination: false,
+                where: {
+                    slug: {
+                        equals: input.category
+                    }
+                }
+            })
+
+            const formatedData = categoriesData.docs.map((doc) => ({
+                ...doc,
+                subcategorise: ((doc as any)?.subcategorise && "docs" in (doc as any).subcategorise ? (doc as any).subcategorise.docs : []).map((subDoc: any) => ({
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)